dashboard/app/models/user.rb

```diff
@@ -484,7 +484,7 @@ def self.find_or_create_facilitator(params, invited_by_user)
   # student.
   has_many :followeds, -> {order 'followers.id'}, class_name: 'Follower', foreign_key: 'student_user_id', dependent: :destroy
   has_many :sections_as_student, through: :followeds, source: :section
-  has_many :teachers, through: :sections_as_student, source: :user
```

Comment:
Nice catch. We may need to search for uses of `:user` as well as `:sections` and add them to the google sheet to change